src/app/fyle/manage-corporate-cards/manage-corporate-cards.page.ts

```
@@ -90,7 +90,7 @@ export class ManageCorporateCardsPage {
 
   segmentChanged(event: SegmentCustomEvent): void {
     if (event.detail.value) {
-      this.segmentValue = parseInt(event.detail.value, 10);
+      this.segmentValue = parseInt(`${event.detail.value}`, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)